IBX-1440: Add description metadata (#133)
ezsystems/user-documentation: docs/personalization/previewing_scenario.md

---

> description: In the Back Office you can preview what results would be recommended by the Personalization service.

Reviewer: ```suggestion
description: In the Back Office you can preview what results are recommended by the Personalization service.
```
